datacapturing/src/main/java/de/cyface/datacapturing/EventHandlingStrategy.java

```diff
@@ -0,0 +1,26 @@
+package de.cyface.datacapturing;
+
+import android.content.Intent;
+import android.os.Parcelable;
+
+import de.cyface.datacapturing.backend.DataCapturingBackgroundService;
+
+/**
+ * Interface for strategies to respond to events triggered by the {@link DataCapturingBackgroundService}.
+ * E.g.: Show a notification when little space is available and stop the capturing.
+ * Must be {@link Parcelable} to be passed from the {@link DataCapturingService} via {@link Intent}.
+ *
+ * @author Armin Schnabel
+ * @version 1.0.0
+ * @since 2.5.0
+ */
```

Comment:
I named this Strategy Interface quite general to avoid frequent API changes on the DataCapturingService constructor. This way we hopefully only have to pass this Strategy class no matter how many events we add to this interface to handle.